packages/comet-uswds/src/components/card/card.figma.tsx

```diff
@@ -1,20 +1,116 @@
-import { Card, CardBody, CardFooter, CardHeader } from './card';
+import { Card, CardBody, CardFooter, CardHeader, CardMedia } from './card';
 import figma from '@figma/code-connect';
 import Button from '../button';
 
+figma.connect(
+  Card,
+  'https://www.figma.com/design/U58Pbb84dLaZfvFvdtGVdT/Comet-UI-Kit?node-id=43-2601',
```

Comment:
URL could be a const

const FIGMA_BASE_URL = 'https://www.figma.com/design/U58Pbb84dLaZfvFvdtGVdT/Comet-UI-Kit?node-id=';

`${FIGMA_BASE_URL}43-2601`,
`${FIGMA_BASE_URL}43-2604`,
etc...

Reply:
You would think, but it actually wouldn’t let me do that. There might be a way, but I couldn’t figure it out.